react/react1/week1/my-app-week1/src/components/Todo.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import TodoList from './TodoList.json'
+export function Po(props) {
```

Comment:
Would suggest putting this component in it's own file. Also think the name Po is not a very clear variable name. Something like Todo or TodoItem might be more descriptive :)